hmvec/hmvec.py

```diff
@@ -429,15 +519,36 @@ def get_power_1halo(self,name="nfw",name2=None):
         ms = self.ms[...,None]
         mnames = self.uk_profiles.keys()
         hnames = self.hods.keys()
-        
-        if (name in mnames) and (name2 in mnames): square_term = self._get_matter(name)*self._get_matter(name2)
-        elif (name in hnames) and (name2 in hnames): square_term = self._get_hod_square(name)
-        elif (name in hnames) and (name2 in mnames): square_term = self._get_hod(name)*self._get_matter(name2)
-        elif (name2 in hnames) and (name in mnames): square_term = self._get_hod(name2)*self._get_matter(name)
-        else: raise ValueError
+        pnames =self.pk_profiles.keys()
+        dampingTerm=1.
+        if (name in hnames) and (name2 in hnames): 
+            square_term = self._get_hod_square(name)
+            dampingTerm *= np.exp(-(self.ks/self.p['kstar_damping'])**2.)
```

Comment:
Why are there multiple factors of the damping term? Why not one overall factor for the whole 1-halo term?

Reply:
After our discussion, I've altered this so all of the terms have the same damping factor (as you had before).